docs/search.md

```diff
@@ -71,7 +71,6 @@ class SearchController extends AbstractController
 {
     use SearchTrait;
 
-    #[Route('/search', name: 'search')]
```

Comment:
i already moved that file to the `cookbook` directory. please rebase your branch